<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Pro_Rec</class>
 <widget class="QMainWindow" name="Pro_Rec">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>719</width>
    <height>518</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pro_Rec</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="show_img">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>10</y>
      <width>448</width>
      <height>448</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>240</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>UnDotum</family>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <property name="text">
     <string>Recgonazition_IMG</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>380</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>UnDotum</family>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <property name="text">
     <string>Type</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>270</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="gparameters">
       <property name="text">
        <string>Get_Parameters</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="recgonization">
       <property name="text">
        <string>Recgonization</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>410</y>
      <width>114</width>
      <height>54</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QRadioButton" name="radioButton_shape">
       <property name="text">
        <string>Shape</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_chalky">
       <property name="text">
        <string>Chalky</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="show_result">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>191</width>
      <height>221</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_8">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLabel" name="label">
             <property name="font">
              <font>
               <family>UnDotum</family>
               <pointsize>15</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="frameShape">
              <enum>QFrame::NoFrame</enum>
             </property>
             <property name="lineWidth">
              <number>3</number>
             </property>
             <property name="text">
              <string>Process_IMG</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="convert">
             <property name="text">
              <string>Convert_IMG</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="denoising">
             <property name="text">
              <string>Denoising</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="splitback">
             <property name="text">
              <string>Split_Back</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <property name="sizeConstraint">
          <enum>QLayout::SetNoConstraint</enum>
         </property>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="layoutDirection">
            <enum>Qt::RightToLeft</enum>
           </property>
           <property name="text">
            <string>(shape)</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>(Chalky)</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="text">
            <string>Boundary</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="histogram">
           <property name="text">
            <string>Histogram</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <zorder>layoutWidget</zorder>
   <zorder>show_img</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>label_4</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>label_5</zorder>
   <zorder>histogram</zorder>
   <zorder>pushButton</zorder>
   <zorder>label_2</zorder>
   <zorder>label_2</zorder>
   <zorder>horizontalLayoutWidget</zorder>
   <zorder>show_result</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>719</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="start">
    <property name="title">
     <string>Start</string>
    </property>
    <addaction name="actionReadFile_2"/>
    <addaction name="actionInitialize"/>
   </widget>
   <addaction name="start"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionReadFile">
   <property name="text">
    <string>ReadFile</string>
   </property>
  </action>
  <action name="actionReadFile_2">
   <property name="text">
    <string>ReadFile</string>
   </property>
  </action>
  <action name="actionset_Type">
   <property name="text">
    <string>set_Type</string>
   </property>
  </action>
  <action name="actionInitialize">
   <property name="text">
    <string>Initialize</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
